app/models/person.rb

```diff
@@ -34,17 +32,41 @@ def currently_matched_with
     current_match.matched_with(self)
   end
 
-  def interest_list
-    interests.map(&:name).join(", ")
+  def to_s
+    name
   end
 
-  def interest_list=(value)
-    self.interests = value.split(",").map(&:strip).uniq.map do |interest|
-      Interest.find_or_initialize_by(name: interest)
+  concerning :Interests do
+    included do
+      has_many :interests, through: :interest_taggings
+      has_many :interest_taggings, dependent: :destroy
+    end
+
+    def interest_list
+      interests.map(&:name).join(", ")
+    end
+
+    def interest_list=(value)
+      self.interests = value.split(",").map(&:strip).uniq.map do |interest|
+        Interest.find_or_initialize_by(name: interest)
+      end
     end
   end
 
-  def to_s
-    name
+  concerning :Professions do
+    included do
+      has_many :professions, through: :profession_taggings
+      has_many :profession_taggings, dependent: :destroy
+    end
```

Comment:
I moved interests and professions to inline concerns, not sure how you feel about that... It's not a lot of code but if it looks confusing we could a) just put the code inline like before or 2) move the code to separate modules.